model/sleep_state.go

```diff
@@ -28,3 +33,24 @@ type SleepState struct {
 type SleepStateTimeout struct {
 	StateExecTimeout StateExecTimeout `json:"stateExecTimeout,omitempty"`
 }
+
+type sleepStateForUnmarshal SleepState
+
+func (s *SleepState) UnmarshalJSON(data []byte) error {
```

Comment:
It's seems useless, can you add a unittest to describe why we need this?

Reply:
indeed, not needed.